python/test/pipeline_test.py

```diff
@@ -450,10 +451,10 @@ def testStartCountdown(self):
   def testStartEta(self):
     """Tests starting a pipeline with an eta."""
     stage = NothingPipeline('one', 'two', three='red', four=1234)
-    eta = datetime.datetime.utcnow() + datetime.timedelta(seconds=30)
+    eta = datetime.datetime.utcnow().replace(tzinfo=pytz.UTC) + datetime.timedelta(seconds=30)
```

Comment:
That is correct. 

Like you said, I'm not sure why we are passing `utcnow()`. In case we want to do that, then we need to add the timezone info to that datetime. If we don't need to do that, we can pass `datetime.now()`. However, whenever we pass a date, it gets converted to UTC internally by the Task class (maybe that's happening locally with the development server). This means that if we pass `datetime.now()` we have to assert this with `utcnow()`. 

Does that make sense?